frameworks/template/universe/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "packagingVersion": "2.0",
-  "minDcosReleaseVersion": "1.9",
+  "packagingVersion": "3.0",
+  "minDcosReleaseVersion": "1.8",
```

Comment:
@ArtRand why was this 1.9 in the first place? 4c0f41248bc72bc028b94d879cfd6c01005fea11